README.md

```diff
@@ -107,120 +88,102 @@ Transaction #2 TransactionOutput {
 }
 ```
 
-3. Check that the genesis block contains 1 pre-configured UTXO for Alice as follows:
-```rust
-utxo::TransactionOutput {
-	value: utxo::Value::max_value(),
-	pubkey: H256::from_slice(get_from_seed::<sr25519::Public>("Alice").as_slice()),
-	salt: 0,
-}
-```
-  - Use the Polkadot JS Chain State app to query the `unspentOutputs` storage map from the `utxoModule`
-  - Query the storage map with the value `0x02f8ef6ff423559b1d07a31075d83c00883fbf02e054af102ab4a0ed8c50d2c5`, which is the hash of the above transaction
-  - You can [use the `subkey` tool to confirm that the public key associated with the initial UTXO belongs to Alice](https://substrate.dev/docs/en/next/development/tools/subkey#well-known-keys)
-
-4. Send a new UTXO transaction from Alice as follows: 
-```rust
-utxo::Transaction {
-	inputs: [ utxo::TransactionInput {
-		parent_output: 0x02f8ef6ff423559b1d07a31075d83c00883fbf02e054af102ab4a0ed8c50d2c5,
-		signature: sig_alice(parent_output)
-	}],
-	outputs: [ utxo::TransactionOutput {
-		value: 100,
-		pubkey: H256::from_slice(get_from_seed::<sr25519::Public>("Bob").as_slice()),
-		salt: 0,
-	}, utxo::TransactionOutput {
-		value: utxo::Value::max_value() - 100,
-		pubkey: H256::from_slice(get_from_seed::<sr25519::Public>("Alice").as_slice()),
-		salt: 0,
-	}]
-}
-```
-  - Use the Polkadot JS Extrinsics app to invoke the `execute` function from the `utxoModule`
-  - You will notice that the transaction's input is the pre-configured UTXO from the genesis block
-  - Use the Polkadot JS Toolbox app to generate the input's `signature` (make sure you are using Alice's account to generate this value)
-  - The public keys are provided for you via console output, which you can easily verify using the `subkey` tool
+6. **Check that Alice already has 100 UTXO at genesis**. In `Chain State` > `Storage`, select `utxoModule`. Input the hash `0x76584168d10a20084082ed80ec71e2a783abbb8dd6eb9d4893b089228498e9ff`. Click the `+` notation to query blockchain state.
```

Comment:
```suggestion
6. **Confirm that Alice already has 100 UTXO at genesis**. In `Chain State` > `Storage`, select `utxoModule`. Input the hash `0x76584168d10a20084082ed80ec71e2a783abbb8dd6eb9d4893b089228498e9ff`. Click the `+` notation to query blockchain state.
```

Just a thought since we already observed Alice's genesis UTXO in the node log.